app/views/events/index.html.erb

```diff
@@ -0,0 +1,52 @@
+<% if @events.upcoming.any? %>
```

Comment:
- [ ] View-related logic can be encapsulated in View helpers (app/helpers/*_helper.rb)

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png)